pkg/tsdb/cloudwatch/cloudwatch.go

```diff
@@ -390,7 +415,7 @@ func (e *cloudWatchExecutor) executeLogAlertQuery(ctx context.Context, req *back
 
 		var frames []*data.Frame
 
-		statsGroups := model.Get("statsGroups").MustStringArray()
+		statsGroups := model.StatsGroups
```

Comment:
No need to copy this value - should be fine to use it directly in the next lines.